frontend/src/pages/Projects/Project/Billing/fields.tsx

```diff
@@ -0,0 +1,46 @@
+import { FormikValues } from "formik";
+import { UseQueryResult } from "react-query";
+import { IEditFields } from "types";
+
+/**
+ * The view fields.
+ *
+ * @param   {UseQueryResult<FormikValues>} reactQuery The react query data for specific project.
+ * @returns {Array}
+ */
+export const readFields = (reactQuery: UseQueryResult<FormikValues>) => {
+  return [
+    {
+      width: "full",
+      title: "Journal Voucher Number",
+      value: reactQuery?.data?.jv_number,
+    },
+    {
+      width: "full",
+      title: "Amount",
+      value: reactQuery?.data?.amount,
+    },
+  ];
+};
+
+/**
+ * The edit fields.
+ *
+ * @returns {Array}
+ */
+export const editFields: () => IEditFields[] = () => {
```

Comment:
This could just be an array rather than a function returning an array but then it would be different than the readFields so I'm not sure that's an improvement. It would just let us define the type as IEditFields[] in the TableData component.